### DISCUSSION TITLE
[Esame 18 ottobre 2024] Esercizio 1

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#605] [Esami M-Z]
![1](https://github.com/user-attachments/assets/5779c71f-65c7-4d83-b8fb-db5ef159f71d)

**Comment by AglaiaNorza** [comment]
![image](https://github.com/user-attachments/assets/c8279e49-0ac7-4aa9-b5d5-deb196fc2262)

**Comment by niccolo-fato** [comment]
<img width="1253" height="640" alt="image" src="https://github.com/user-attachments/assets/5a744983-59b1-4a7a-92f6-39dc624a5082" />
